public/components/agents/vuls/inventory.tsx

```diff
@@ -0,0 +1,218 @@
+/*
+ * Wazuh app - Agent vulnerabilities components
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React, { Component } from 'react';
+import {
+  EuiPanel,
+  EuiPage,
+  EuiTitle,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiProgress,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty
+} from '@elastic/eui';
+import {
+  InventoryTable,
+  FilterBar
+} from './inventory/';
+import { WzRequest } from '../../../react-services/wz-request';
+import exportCsv from '../../../react-services/wz-csv';
+import { getToasts }  from '../../../kibana-services';
+import { ICustomBadges } from '../../wz-search-bar/components';
+import { filtersToObject } from '../../wz-search-bar';
+
+export class Inventory extends Component {
+  _isMount = false;
+  state: {
+    filters: [];
+    totalItems: number;
+    isLoading: Boolean;
+    items: [];
+    customBadges: ICustomBadges[];
+  };
+
+  props: any;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      filters: [],
+      items: [],
+      totalItems: 0,
+      isLoading: true,
+      customBadges: []
+    }
+    this.onFiltersChange.bind(this);
+  }
+
+  async componentDidMount() {
+    this._isMount = true;
+    await this.loadAgent();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (JSON.stringify(this.props.agent) !== JSON.stringify(prevProps.agent)){
+      this.setState({isLoading: true}, this.loadAgent)
+    }
+  }
+
+  componentWillUnmount() {
+    this._isMount = false;
+  }
+
+  async loadAgent() {
+    const {totalItems, items} = await this.getItemNumber();
+    if (this._isMount){
+      this.setState({ totalItems, items, isLoading: false });
+    }
+  }
+
+  getStoreFilters(props) {
+    const { section, selectView, agent } = props;
+    const filters = JSON.parse(window.localStorage.getItem(`wazuh-${section}-${selectView}-vulnerability-${agent['id']}`) || '{}');
+    return filters;
+  }
+
+  setStoreFilters(filters) {
+    const { section, selectView, agent } = this.props;
+    window.localStorage.setItem(`wazuh-${section}-${selectView}-vulnerability-${agent['id']}`, JSON.stringify(filters))
+  }
+
+  onFiltersChange = (filters) => {
+    this.setStoreFilters(filters);
+    this.setState({ filters });
+  }
+
+  onTotalItemsChange = (totalItems: number) => {
+    this.setState({ totalItemsFile: totalItems });
+  }
+
+  buildFilter() {
+    const filters = filtersToObject(this.state.filters);
+    const filter = {
+      ...filters,
+      limit: '15' 
```

Comment:
Why the limit is fixed?

Reply:
Because is just the default value. The page size value can be selected within the table component.

Reply:
But if the page size changes, the `limit` variable continues on `15`, no?